wagtail/images/fields.py

```diff
@@ -186,8 +186,6 @@ def to_python(self, data):
 def image_format_name_to_content_type(image_format_name):
     """
     Convert a Willow image format name to a content type.
-    TODO: Replace once https://github.com/wagtail/Willow/pull/102 and
-          a new Willow release is out
```

Comment:
Thanks, but the point is not just about removing the TODO comment. It says that this should be **replaced** once there's a new Willow release that includes the changes in that PR, which also means we need to ensure the minimum Willow version supported by Wagtail is that release's version or greater. If the condition is fulfilled, then we need to replace this function in favour of Willow's `mime_type` property.

Reply:
The current version of willow is `v1.8` ?

Reply:
Yes, and that PR was included since v1.5.

It seems Wagtail already requires Willow v1.8 and up:
https://github.com/wagtail/wagtail/blob/9d1bf256a276e9508bf75ce7e3696468646718f0/setup.py#L32

That means we can proceed with removing the function and replacing its usage with Willow's `mime_type` property.

Reply:
Apologies for delay, pushed required changes.